main.go

```diff
@@ -29,62 +30,32 @@ func main() {
 	}
 
 	var config config.Config
-	err = cleanenv.ReadConfig("config.json", &config)
+	err = cleanenv.ReadConfig(configPath, &config)
 	if err != nil {
 		log.Fatal(fmt.Errorf("read config: %w", err))
 	}
 
 	logger, _ := zap.NewProduction()
 
-	db, err := sql.Open(env.SqlDriver, env.Connection)
+	db, err := sql.Open("pgx", env.ConnectionStr())
 	if err != nil {
 		log.Fatal(fmt.Errorf("creating db: %w", err))
 	}
 	err = db.Ping()
 	if err != nil {
 		db.Close()
-		log.Fatal(fmt.Errorf("ping db %s : %w", env.Connection, err))
+		log.Fatal(fmt.Errorf("ping db %s : %w", env.ConnectionStr(), err))
 	}
 
-	migrationsPath := "migrations"
-	err = postgres.MigrateUp(migrationsPath, env.Connection)
+	err = postgres.MigrateUp(migrationsPath, env.ConnectionStr())
 	if err != nil {
 		log.Fatal(fmt.Errorf("db migrations: %w", err))
 	}
 
-	initTestData(db)
+	test.InitTestData(db)
```

Comment:
pls remove seeds from main function,error logs cause a lot of confusions

Reply:
@igkostyuk I don't understand what do you mean by that

Reply:
@glbter every time on running main it seeds initial users, on the second run and so on its produce  Postgres error in DB container because users already exist, this error confusing  out co-workers